### PR TITLE
update in-repo api documentation with info about the decision review api

### PR DESCRIPTION
**Pull request by edmangimelli**
### Description
Update `caseflow-apis.md` with info about the Decision Review API

### Acceptance Criteria
- [ ] provides the correct amount of detail about the Decision Review API

### Note
compare the old documentation:
https://github.com/department-of-veterans-affairs/caseflow/blob/66b29d22211b406b90c416d1c5252a602722b251/docs/caseflow-apis.md
to the new:
https://github.com/department-of-veterans-affairs/caseflow/blob/ed666ef556adf2d843a89f8af431c0cfff99e99b/docs/caseflow-apis.md

### CODE REVIEW

File: docs/caseflow-apis.md [hunk 1]
```diff
@@ -351,4 +359,102 @@ On success returns `200` HTTP code and a body like:
...
+
+**Note:** The curl examples above expect completely different API keys. Watch out for this.
+
+#### Implementation Notes
+
+The Decision Review API depends heavily on the existing controllers/models/classes that support Caseflow Intake, but diverges where necessary to support the different audience of the API. The Decision Review API is actively being developed with an MVP of supporting a va.gov front end for veterans to submit their own decision reviews.
+
+##### JSON:API
+
+In keeping with other APIs associated with the Lighthouse project, the Decision Review API returns JSON conforming to the [JSON:API ](https://jsonapi.org/format/) specification (v1.0).
```
Comment (erluti) [inline docs/caseflow-apis.md]: Also the V2 API uses JSONApi, so I think it's okay to just say it's JSONApi without blaming Lighthouse, lol

**Comment by pkarman** [comment]
This document has moved to https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-APIs so you can just update the wiki directly.

**Comment by edmangimelli** [comment]
> This document has moved to https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-APIs so you can just update the wiki directly.

@pkarman
diffing master's `caseflow-apis.md` and the wiki's `Caseflow-APIs.md` revealed that they were the same, so I just copied over my updates to the wiki manually.
https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-APIs#caseflow-jobs
(^^^ updated to reflect the changes in this PR)

Should we still merge this PR (that is, update the _in-repo_ documentation)? or call it good?

**Comment by pkarman** [comment]
No, we're going to pull the .md file and just point at the wiki.

**Comment by edmangimelli** [comment]
>No, we're going to pull the .md file and just point at the wiki.

**Pull request closed without merging**